documentation/grammars/en_toy/classify/email.grm

```diff
@@ -0,0 +1,51 @@
+import '../byte.grm' as b;
+import '../util.grm' as u;
+
+d = b.kDigit;
+q = u.q;
+
+
+symbols = "!" | "#" | "$" | "%" | "&" | "'" | "*" | "+" | "-" | "/" | "=" | "?" | "^" | "_" | "`" | "{" | "|" | "}" | "~";
+
+
+#email_id_digits = b.kAlnum | symbols;
+
+email_id_digits = b.kGraph - "@";
+domain_digits = b.kGraph - ".";
+
+#email =
+#    u.I["decimal { "]
```

Comment:
Can you please add to this and other .grm files something like this: #
# Jan. 3,? 1980
#
# or
#
# 3 Jan.,? 1980
#
# into
#
# date { month: "January" day: "3" year: "1980" }